core/controllers/library.py

```diff
@@ -300,7 +300,8 @@ def get(self):
             summaries = (
                 summary_services.get_displayable_exp_summary_dicts_matching_ids(
                     exp_ids,
-                    editor_user_id=editor_user_id))
+                    editor_user_id=editor_user_id,
+                    editor_actions=self.actions))
```

Comment:
There are many cases where this function is called only with exp_ids. If we force to pass args explicitly.
Then call will look like, get_displayable_exp_summary_dicts_matching_ids(exp_ids, None, None). 
 Should I proceed with this?